src/stories/button/Button.tsx

```diff
@@ -46,12 +57,19 @@ export type ButtonProps = {
      * The size of the Button.
      * @default large
      */
-    size?: 'large' | 'small' | 'xs';
+    size?: 'large' | 'medium' | 'small' | 'xs';
     /**
      * The shape of the Button.
      * @default pill
      */
-    shape?: 'pill' | 'circle' | 'rectangle' | 'square' | 'rounded';
+    shape?: 'pill' | 'circle' | 'rectangle' | 'square';
+    /**
+     * The radius of the corners for the `rectangle` and `square` Button shapes. Either a preset or a valid {@link CSSLength} string.
+     * `sharp` will have no rounding, `rounded` will have a slight rounding, and `roundedXL` will have a large rounding.
+     * @see CornerPresets
+     * @default rounded
+     */
+    corners?: typeof CornerPresets[number] | CSSLength;
```

Comment:
~~accepts number, not `CSSLength`?~~

never mind, seems like you did built it to accept `CSSLength` but Storybook just isn't handling properly for now. let's add a ticket to fix this at some point, probably will just need to manually define the prop in the story